config/routes.rb

```diff
@@ -73,6 +73,8 @@
 
   get 'orga/users/info', to: 'orga/users_info#index', as: :orga_users_info
   patch 'orga/seasons/switch_phase', to: 'orga/seasons#switch_phase', as: :switch_phase
+  put 'orga/community/reset_user_availability', to: 'orga/community#reset_user_availability'
+  put 'users/current/update_availability', to: 'users#set_available'
```

Comment:
Can you please move this to the `users` resource and make it a member action? It's really hard to find otherwise, given the size of the `routes` file 🙈